lib/utils/__tests__/isStandardSyntaxFunction.test.js

```diff
@@ -6,38 +6,32 @@ const valueParser = require('postcss-value-parser');
 
 describe('isStandardSyntaxFunction', () => {
 	it('calc', () => {
-		funcs('a { prop: calc(a + b) }', (func) => {
-			expect(isStandardSyntaxFunction(func)).toBeTruthy();
-		});
+		expect(isStandardSyntaxFunction(func('a { prop: calc(a + b) }'))).toBeTruthy();
```

Comment:
```suggestion
		expect(isStandardSyntaxFunction(func('a { prop: calc(a + b) }'))).toBe(true);
```

Can we use `toBe(true)` and `toBe(false)` in this file as they are more strict comparison than [`toBeTruthy`](https://jestjs.io/docs/en/expect#tobetruthy)/`toBeFalsy`.